airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -325,6 +325,13 @@
   documentationUrl: https://docs.airbyte.io/integrations/sources/db2
   icon: db2.svg
   sourceType: database
+- name: Infinicept
+  sourceDefinitionId: 11856c6d-37c3-4cdf-81c0-f1fe2fbaf280
+  dockerRepository: airbyte/source-infinicept
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.io/integrations/sources/infinicept
```

Comment:
Should we point to out own confluence maybe, since this link isn't live?

Reply:
Yup agreed. Alternatively we can also document in this connector README and link to that. Either should hopefully work